### PR TITLE
Tourettes stun duration reduced from 10 seconds to 2 seconds

### PR DESCRIPTION
**Pull request by Jujumatic**
After managing to somehow decontaminate the master branch for long enough to put this, it's finally here. I've learned branch management the hard way.

Stun time for Tourettes has been reduced from 10 seconds to 2 seconds, because these poor fucking souls are already suffering enough if they have Tourettes to begin with. The stun spam is already bad enough yo

### CODE REVIEW

**Comment by JohnGinnane** [comment]
hhhhhhhhhhhhholy fuck thank you

File: hippiestation/code/datums/traits/neutral.dm [hunk 1]
```diff
@@ -1,2 +1,2 @@
 /* Hippie Neutral Traits */
-/* Currently, there are none */
\ No newline at end of file
+-/* Currently, there are none */
```
Comment (JohnGinnane) [inline hippiestation/code/datums/traits/neutral.dm]: this dash shouldn't be here

**Review by JohnGinnane — CHANGES_REQUESTED** [review]

File: hippiestation/code/datums/traits/neutral.dm (hunk 1 above)
Comment (Jujumatic) [reply hippiestation/code/datums/traits/neutral.dm]: how did that get there lmao

**Comment by Jujumatic** [comment]
Is fixed aaaaaaaa

**Comment by Pyko1** [comment]
sure why not

**Comment by JohnGinnane** [comment]
oops, didn't mean to add that tag

do this need head input as it's maybe balance? @HippieStation/head

**Comment by JohnGinnane** [comment]
I think I broke this PR, would you mind doing it up again?

**Comment by ghost** [comment]
Good work bro

**Comment by ghost** [comment]
To juju for his PR I mean, proud of u ducky boy

**Comment by Jujumatic** [comment]
oof fuck what happened

**Comment by Jujumatic** [comment]
also ty dude

**Comment by Jujumatic** [comment]
Unfucked everything this should be ready to merge now

can i go to bed now :'(

**Review by JohnGinnane — APPROVED** [review]

**Pull request merged**